webapp/components/search_results_item.jsx

```diff
@@ -187,7 +186,7 @@ export default class SearchResultsItem extends React.Component {
         let message;
         let flagContent;
         let rhsControls;
-        if (post.state === Posts.POST_DELETED) {
+        if (post.state === Constants.POST_DELETED) {
```

Comment:
The redux one is capitalized. In RN do you just use the constants from mattermost-redux? If so then we can just change it in redux and everything should just work